spec/models/rr_pending_change_spec.rb

```diff
@@ -32,4 +32,25 @@
       end
     end
   end
+
+  describe ".backlog_details" do
+    it "returns the correct counts" do
+      MiqRegion.seed
+      region = MiqRegion.my_region_number
+
+      ActiveRecord::Schema.define do
+        create_table "rr#{region}_pending_changes" do |t|
+          t.string    :change_table
+          t.string    :change_key
+          t.string    :change_new_key
+          t.string    :change_type
+          t.timestamp :change_time
+        end
+      end
+      FactoryGirl.create(:rr_pending_change, :change_table => "users")
+      FactoryGirl.create(:rr_pending_change, :change_table => "miq_servers")
```

Comment:
you could possibly use `FactoryGirl.create_list` here to remove the duplicate line below